doc/github-actions.md

```diff
@@ -0,0 +1,26 @@
+# GitHub Actions
+
+## Configuration
+
+his project supports github actions for continuous integration and delivery. To enable GitHub actions on this repo, you will
+have to create the following secrets:
+
+```bash
+SUPABASE_ACCESS_TOKEN: Your personal access token, can be found at https://supabase.com/dashboard/account/tokens
+SUPABASE_DB_PASSWORD: Your supabase database password
+SUPABASE_PROJECT_ID: Your supabase project id
+SUPABASE_URL: Your supabase project URL
+SUPABASE_ANON_KEY: Your supabase project anonymous key
+```
+
+Also, you will need to configure the some variables:
+```bash
+VITE_IS_DEMO: Set to `true` if you want to display the demo banner
+```
+
+## Deploying to Another Repository
```

Comment:
nitpick: Same. We have this need internally because we want to deploy both the landing page and the demo using gh-pages, but our users probably won't have this need. I'm not sure we need to document this variable.

Reply:
I'm not sure if users will want to push to another repository (or even if they will want to push to GH pages), but this does not cost anything to document it